landlab/grid/landslide_unitgrid.py

```diff
@@ -0,0 +1,540 @@
+"""
+Unit tests for landlab.components.landslides.landslide_probability
+"""
+import numpy as np
```

Comment:
@ChristinaB I don't think this file's contents should be located in landlab/grid. I think it would be better suited for the conftest.py file you'v created. I'm interested in @mcflugen's thoughts on this.